open-api-spec.yaml

```diff
@@ -838,7 +838,8 @@ components:
           readOnly: true
         sharedSecret:
           type: string
-          description: The shared secret to use when establishing a STREAM connection.
+          pattern: "^[a-zA-Z0-9-_]{43}$"
```

Comment:
The [STREAM spec](https://interledger.org/rfcs/0029-stream/#4-life-of-a-connection):
> Cryptographically secure random or pseudorandom shared secret (it is RECOMMENDED to use 32 bytes)

Should we not enforce the length here if 32 bytes technically isn't `REQUIRED`?
fwiw, ilp-pay has been enforcing 32-bytes:
https://github.com/interledgerjs/interledgerjs/blob/b3817e441f0c797cfd094bdbb5e4d6d046893613/packages/pay/src/open-payments.ts#L11
https://github.com/interledgerjs/interledgerjs/blob/b3817e441f0c797cfd094bdbb5e4d6d046893613/packages/pay/src/open-payments.ts#L276-L286

Reply:
I'm leaning towards not enforcing it if it's not required by the STREAM spec. ilp-pay is just an implementation that chooses to enforce it and our implementation can also enforce it. But I wouldn't enforce it in this spec.